server/db/models/user.js

```diff
@@ -6,10 +6,13 @@ const User = db.define('user', {
   email: {
     type: Sequelize.STRING,
     unique: true,
-    allowNull: false
+    allowNull: false,
+    validate: {
+      isEmail: true
```

Comment:
Add notEmpty validation? I don't really know what the difference between allowNull vs notEmpty is though.

Reply:
I'll cut out allowNull. It is redundant